guides/04_additional-features/07_sharing-your-app.md

```diff
@@ -53,6 +54,18 @@ After you have [created a free Hugging Face account](https://huggingface.co/join
   <source src="https://github.com/gradio-app/gradio/blob/main/guides/assets/hf_demo.mp4?raw=true" type="video/mp4" />
 </video>
 
+## Sharing Deep Links
+
+You can add a button to your Gradio app that creates a unique URL you can use to share your app **as it currently is** with others. This is useful for sharing unique and interesting generations from your application , or for saving a snapshot of your app at a particular point in time.
+
+To add a deep link button to your app, place the `gr.DeepLinkButton` component anywhere in your app.
+For the URL to be accessible to others, your app must be available at a public URL. So be sure to host your app like Hugging Face Spaces or use the `share=True` parameter when launching your app.
+
+Let's see an example of how this works. Here's a simple Gradio chat ap that uses the `gr.DeepLinkButton` component. After a couple of messages, click the deep link button and paste it into a new browser tab to see the app as it is at that point in time.
+
+$code_deep_link
+$demo_deep_link
```

Comment:
I don't think this demo was pushed (which might also be why the CI is failing)

Reply:
yea good call!